_posts/2021-12-16-ngff.md

```diff
@@ -0,0 +1,142 @@
+---
+layout: post
+title: OME-NGFF 2021 updates
+tags: file-formats community
+categories: blog
+---
+
+In December 2020, we published two blog posts reporting on the development of
+next-generation file formats (NGFFs). We discussed public specifications for
+storing [multiscale images, label images]({{ site.baseurl }}{% post_url 2020-11-04-zarr-data %})
+and [high-content screening datasets]({{ site.baseurl }}{% post_url 2020-12-01-zarr-hcs %}).
+
+Here we report the progress made on NGFFs over the course of 2021 with contributions from OME, Glencoe Software, EMBL, Harvard Medical School.
+
+
+## NGFF paper
+
+The principle of next-generation file formats as a solution for bioimaging
+data storage and access was recently published as a
+[Brief Communication in Nature Methods](https://www.nature.com/articles/s41592-021-01326-w).
+This work was driven by OME and Glencoe Software, with significant
```

Comment:
maybe rather than repeating the list here, we could add to the previous listing, "...all of whom contributed to the NGFF paper".